---
title: "Bacteria found on the Cellphone Screens of Students vs. Professors"
author: "Shreya Barma"
date: "August 27, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

This project looks at the different types and amounts of bacteria on the cell phone screens of students compared to the amount of bacteria found on the cell phone screens of professors. This is to see whether or not phones carry potential pathogens, which are disease-causing bacteria. Thus, this experiment will help build a correlation between how illnesses are spread and technology usage.I will specifically focus on the presence of E.Coli, Staphylococcus, and Pseudomonas as they are the most common pathogens found on cellphones. 

This has been shown through studies in the past with teenagers and their cell phones, "We found a high median bacterial count on secondary school studentsâ€™ mobile phones...and a median of 17,032 bacterial 16S rRNA gene copies per phone. Potentially pathogenic microbes were found among dominant microbes more often on phones with higher percentage of E. faecalis in total bacterial 16S rRNA" [@cogprints6566]. 

Furthermore, another study proved that there is an extremely high amount of bacteria on cellphones. "In total, 94.5% of phones demonstrated evidence of bacterial contamination with different types of bacteria...S. aureus strains isolated from mobile phones of 52% and those strains isolated from hands of 37.7% were methicillin resistant" [@ulger2009we]




# Question

"Do cellphones really carry potential pathogens? Does age play a factor in cellphone hygeine?" 

# Hypothesis 

All cellphones will carry potential pathogens. Student cellphones will have a significantly higher amount of potential pathogens compared to professor cellphones. 

## Methods

For this project, I collected a total of twelve samples, two from each cellphone. 
Therefore, I swabbed a total of six phones. To collect my samples, I used a sterile 
swab dipped in a buffer. I swabbed the entire front screen of each phone for 30 
seconds and then stored the swab in a sterile tube. Once I finished collecting 
all my samples, I placed all of them 

### DNA Extraction 
For DNA extraction, I followed the manufacturer's protocol from the Sigma REDExtract-N-Amp Kit. 

### PCR
To make the master mix for the PCR reaction, I used the formula: (n+1)+10%, where n is the number of samples I have, which in this case is six. With all the calculations, the master mix included 77 microliters of Amp, 6.2 microliters of the 27f primer, 6.2 microliters of 1492r primer, 7.7 microliters of BSA, and 49.28mL of PCR water. For each alloquat, I added 19 microliters of the master mix and 1 microliter of the specific DNA template. 




## Analysis

# Results

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

| description                                                                       | max score | total score | query cover | E value   | per. identity | Accession  |
|-----------------------------------------------------------------------------------|-----------|-------------|-------------|-----------|---------------|------------|
| Kocuria palustris strain SR5-19 16S ribosomal RNA gene, partial sequence          | 1129      | 1129        | 100%        | 0.0       | 99.20%        | MN421514.1 |
| Staphylococcus sp. strain 21 16S ribosomal RNA gene, partial sequence             | 931       | 931         | 100%        | 0.0       | 99.80%        | MK465362.1 |
| Staphylococcus epidermidis strain IBK-11 16S ribosomal RNA gene, partial sequence | 473       | 473         | 100%        | 1x10^-129 | 98.88%        | MN428237.1 |


```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data-Sheet1.csv")
```

```{r filter-and-plot-abundances}
# filter out only 100x dilutions and make boxplot
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of Colony Abundances Between Student
          & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances from Professor and Student cellphones, 1x dilution. Despite a higher median number of colonies from student samples, the mean values were not significantly different between the two sites (Wilcox p = 1). 

```{r abundances-stats}
#run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = . ) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only 100x dilutions and make boxplot
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Morphotypes Between Student & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```


**Figure 2:**Boxplot showing the number of morphotypes from student cellphones compared to professor cellphones. Ther was no difference in the mean number of morphotypes (Wilcox p=1).

```{r morphotypes-stats}
#run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes)~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```
  

## Culture Data

# Discussion

# Sources Cited
```{r your-code-here}
# Add code chunks as needed for your analyses
```

# Discussion

# Sources Cited
