---
title: "Different Bacteria found on the Screens of Cellphones of Students vs. Professors"
author: "Shreya Barma"
date: "August 27, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Overview of Project 

The project I am looking at will look at the amount of bacteria on the cell phone screens of students and comparing them the amount of bacteria found on the cell phone screens of professors. This is to see whether or not phones carry potential pathogens or disease-causing bacteria. Thus, this experiment can help build a correlation between how illnesses are spread and technology usage.I will specifically focus on the presence of E.Coli, Staphylococcus, and Pseudomonas.

This has been shown through studies in the past with teenagers and their cell phones, "We found a high median bacterial count on secondary school studentsâ€™ mobile phones...and a median of 17,032 bacterial 16S rRNA gene copies per phone. Potentially pathogenic microbes were found among dominant microbes more often on phones with higher percentage of E. faecalis in total bacterial 16S rRNA" [@cogprints6566]. 

Furthermore, another study proved that there is an extremely high amount of bacteria on cellphones. "In total, 94.5% of phones demonstrated evidence of bacterial contamination with different types of bacteria...S. aureus strains isolated from mobile phones of 52% and those strains isolated from hands of 37.7% were methicillin resistant" [@ulger2009we]

# Introduction

# Question

"Do cellphones really carry potential pathogens? Does age play a factor in cellphone hygeine?" 

# Hypothesis 

All cellphones will carry potential pathogens. Student cellphones will have a significantly higher amount of potential pathogens compared to professor cellphones. 

## Methods


## Study Design

For my sample set I will collect swabs from 3 students' cellphones and also collect swabs from 4 teachers. These students will be two of my housemates and myself, and the professors will be from the classes I am currently taking. Once I have completed collecting my sample sets, I will extract DNA and culture the bacteria, identify specific types of bacteria I am focusing the study on, and run PCR's. 

## Materials Needed 

* sterile swabs 
* sterile tubes 
* sterile buffer  

## Analysis

# Results

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

| description                                                                       | max score | total score | query cover | E value   | per. identity | Accession  |
|-----------------------------------------------------------------------------------|-----------|-------------|-------------|-----------|---------------|------------|
| Kocuria palustris strain SR5-19 16S ribosomal RNA gene, partial sequence          | 1129      | 1129        | 100%        | 0.0       | 99.20%        | MN421514.1 |
| Staphylococcus sp. strain 21 16S ribosomal RNA gene, partial sequence             | 931       | 931         | 100%        | 0.0       | 99.80%        | MK465362.1 |
| Staphylococcus epidermidis strain IBK-11 16S ribosomal RNA gene, partial sequence | 473       | 473         | 100%        | 1x10^-129 | 98.88%        | MN428237.1 |


```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv("data/raw_data/raw_culture_count_data/BIOL422_culturing_data - Sheet1.csv")
```

```{r filter-and-plot-abundances}
# filter out only 100x dilutions and make boxplot
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of Colony Abundances Between Student
          & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances from Professor and Student cellphones, 1x dilution. Despite a higher median number of colonies from student samples, the mean values were not significantly different between the two sites (Wilcox p = 1). 

```{r abundances-stats}
#run a statistical test to compare the two groups of abundances 
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
   broom::glance(x = . ) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only 100x dilutions and make boxplot
culture_data %>% 
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>% 
  ggplot(aes(x=sample_group,
             y=as.numeric(number_of_morphotypes))) +
  geom_boxplot() + 
  ggtitle("Comparison of Morphotypes Between Student & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```


**Figure 2:**Boxplot showing the number of morphotypes from student cellphones compared to professor cellphones. Ther was no difference in the mean number of morphotypes (Wilcox p=1).

```{r morphotypes-stats}
#run a statistical test to compare the two groups of abundances 
culture_data %>% 
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>% 
  wilcox.test(data = ., 
              as.numeric(number_of_morphotypes)~ sample_group) %>%
  broom::glance(x = . ) %>%
  knitr::kable()

```
  

## Culture Data

# Discussion

# Sources Cited
```{r your-code-here}
# Add code chunks as needed for your analyses
```

# Discussion

# Sources Cited
