---
title: "Bacteria found on the Cellphone Screens of Students vs. Professors"
author: "Shreya Barma"
date: "August 27, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

The cellphone has become a staple in today's society. It's role has drastically 
expanded from the typical use of a phone, to call or text someone. It's now used
as a GPS device, to browse the internet, and much more. Due to all of these 
applications, cellphones are being used all most all day, everyday by their 
owners. This daily use and the benefits provided by a cellphone, causes 
individuals to overlook cellphone hygeine, or even consider the amount of
bacteria that accumulates on the screen of a phone. 

This project focuses on identifying the type and amount bacteria on cell phone
screens of students at the University of San Francisco compared to those found 
on the cellphone screens of professors. This is to see whether or not phones 
carry potential pathogens, which are disease-causing bacteria. Thus, this 
experiment will help build a correlation between how illnesses are spread and 
technology usage.I will specifically focus on the presence of *E.Coli*, 
*Staphylococcus*, and *Pseudomonas* as they are the most common pathogens found on 
cellphones. 

Studies have been done in clinical settings, showing how healthcare 
professionals use their phones throughout the day, both before and after 
assessing patients. This resulted in a significant transfer of bacteria, since 
most cellphone users, healthcare workers included, do not clean their cellphone 
screens. "Different studies in different parts of the world indicated that 
medical equipment and mobile phones of healthcare workers are a potential source
of nosocomial infections" (add citation). This puts not only cellphone users at 
risk, but the patients, the health care workers and their families as well. 

However, healthcare workers are not the only ones with phones carrying bacteria, 
studies have shown that with a general cellphone user, there is 10x more 
bacteria on a cellphone screen than a toilet seat. Aside from clinical settings, 
I aimed to focus on a college setting. A study on the bacterial contamination of 
middle school students' cellphones concluded that teenagers have "...a median of 17,032 bacterial 16S rRNA gene copies per phone" [@cogprints6566]. Furthermore,
out of the number of bacteria found on these phones, a significant number of 
were labeled "potentially pathogenic". These bacteria included *E. faecalis*

Furthermore, another study proved that there is an extremely high amount of 
bacteria on cellphones. "In total, 94.5% of phones demonstrated evidence of 
bacterial contamination with different types of bacteria...S. aureus strains 
isolated from mobile phones of 52% and those strains isolated from hands of 
37.7% were methicillin resistant" [@ulger2009we]


# Question

"Do cellphones really carry potential pathogens? Does age play a factor in 
cellphone hygeine?" 

# Hypothesis 

All cellphones will carry potential pathogens. Student cellphones will have a 
significantly higher amount of potential pathogens compared to professor 
cellphones. 

## Methods

For this project, I collected a total of twelve samples, two samples per 
cellphone. Therefore, I swabbed a total of six phones. To collect my samples, 
I used a sterile swab dipped in a buffer. I swabbed the entire front screen of 
each phone for 30 seconds and then stored the swab in a sterile tube. Once I 
finished collecting all my samples, I placed all of them 

### DNA Extraction 
For DNA extraction, I followed the manufacturer's protocol from the Sigma 
REDExtract-N-Amp Kit. 

### PCR
The gel for the PCR was a standard 1% agarose gel and was prepared ahead of 
time by Professor Zimmerman. To prepare the gel, 1g of agarose is measured and 
mixed with 100mL of 1xTAE buffer in a microwaveable flask. This flask is then 
microwaved for 1-3 minutes until the agarose powder is completely dissolved. 
Then the agarose and buffer mixture is allowed to cool to the point where it is 
comfortable to hold the flask. It is then poured into the gel tray with the well
comb already set in place. Once poured, the tray with the agarose gel is set out
to cool for around 10 minutes or until fully hardened. 

To make the master mix for the PCR reaction, I used the formula: (n+1)+10%, 
where n is the number of samples I have, which in this case is six. With all 
the calculations, the master mix included 77 microliters of Amp, 6.2 microliters
of the 27f primer, 6.2 microliters of 1492r primer, 7.7 microliters of BSA, and 
49.28mL of PCR water. For each alloquat, I added 19 microliters of the master 
mix and 1 microliter of the specific DNA template.

These tubes were then taken into the thermocycler by Professor Zimmerman and 
set to run at the following temperatures:

| Temperature (°C) | Duration   |
|------------------|------------|
| 95°C             | 5 minutes  |
| 94°C             | 30 seconds |
| 65°C             | 30 seconds |
| 72°C             | 1 minute   |
| 94°C             | 30 seconds |
| 55°C             | 30 seconds |
| 72°C             | 1 minute   |
| 72°C             | 10 minutes |
| 4°C              | hold       |

Once the thermocycling was complete, the gel was carefully loaded with 2-20 
microliter pipettes and then run on the electrode for 30 minutes. 
"After checking for successful amplification using gel electrophoresis, PCR products were cleaned with Invitrogen ExoSAP and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA)." 




## Analysis

# Results

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

| description                                                                       | max score | total score | query cover | E value   | per. identity | Accession  |
|-----------------------------------------------------------------------------------|-----------|-------------|-------------|-----------|---------------|------------|
| Kocuria palustris strain SR5-19 16S ribosomal RNA gene, partial sequence          | 1129      | 1129        | 100%        | 0.0       | 99.20%        | MN421514.1 |
| Staphylococcus sp. strain 21 16S ribosomal RNA gene, partial sequence             | 931       | 931         | 100%        | 0.0       | 99.80%        | MK465362.1 |
| Staphylococcus epidermidis strain IBK-11 16S ribosomal RNA gene, partial sequence | 473       | 473         | 100%        | 1x10^-129 | 98.88%        | MN428237.1 |


```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data-Sheet1.csv")
```

```{r filter-and-plot-abundances}
# filter out only 100x dilutions and make boxplot
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of Colony Abundances Between Student
          & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances from Professor and Student cellphones, 1x dilution. Despite a higher median number of colonies from student samples, the mean values were not significantly different between the two sites (Wilcox p = 1). 

```{r abundances-stats}
#run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = . ) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only 100x dilutions and make boxplot
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle(
    "Comparison of Morphotypes Between Student & Professor Cellphones", "1x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```


**Figure 2:**Boxplot showing the number of morphotypes from student cellphones compared to professor cellphones. Ther was no difference in the mean number of morphotypes (Wilcox p=1).

```{r morphotypes-stats}
#run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Shreya Barma") %>%
  filter(dilution_1_10_or_100 == "1") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes)~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```
  

## Culture Data

# Discussion

# Sources Cited
```{r your-code-here}
# Add code chunks as needed for your analyses
```

# Discussion

# Sources Cited
